docs-site/bundle.js

```diff
@@ -36864,13 +36864,12 @@
 	      //  If the above moment.set() call returned a valid date, publish the new date object:
 	      if (changedDate && changedDate.isValid()) {
 	        this.props.onChange(changedDate, event);
-	      }
 
-	      //  Else the date *isn't* valid, but if we allow invalid dates to be entered
-	      //  anyway, publish the change using a null value:
-	      else if (this.props.allowInvalidDates) {
-	          this.props.onChange(null, event);
-	        }
+	        //  Else the date *isn't* valid, but if we allow invalid dates to be entered
```

Comment:
These are all changes to the output bundle.js file and can be ignored; the _real_ change is in the /src files.